command/s3_put.go

```diff
@@ -299,9 +301,11 @@ retryLoop:
 						if s3pc.isMulti() {
 							// try the remaining multi uploads in the group, effectively ignoring this
 							// error.
+							notFound++
```

Comment:
I'm not convinced we will ever increment this value, since `util.BuildFileList` should presumably always return files that exist. Instead we want to log how many files we *do* upload, and potentially log the special case that `util.BuildFileList` found nothing.